api/SpeechGrammarList.json

```diff
@@ -0,0 +1,415 @@
+{
+  "api": {
+    "SpeechGrammarList": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SpeechGrammarList",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "prefix": "webkit",
+            "version_added": "33",
+            "notes": "You'll need to serve your code through a web server for recognition to work."
+          },
+          "chrome_android": {
+            "prefix": "webkit",
+            "version_added": true,
+            "notes": "You'll need to serve your code through a web server for recognition to work."
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "44",
+            "flags": [
+              {
+                "type": "preference",
+                "name": "media.webspeech.recognition.enable"
+              }
+            ],
+            "notes": "currently speech recognition won't work on Desktop Firefox — it will be properly exposed soon, once the required internal permissions are sorted out."
```

Comment:
This is a weird situation, where there is a related preference, but ultimately web developers can't use the API regardless. For each of these "currently speech recognition won't work on Desktop Firefox" notes, I think the right thing to do here is go with `"version_added": false` and drop everything else.

Reply:
I agree, this is probably still from Firefox OS times. I changed this to just false.